src/app/shared/components/join-us/join-us.component.html

```diff
@@ -2,20 +2,19 @@
   <div class="join">
     <div class="info">
       <div class="letter">NEWS LETTER</div>
-      <div class="message">join us now to get all news and special offers</div>
+      <div class="join-message">join us now to get all news and special offers</div>
     </div>
     <div class="register">
-      <form [formGroup]="submitForm" (ngSubmit)="onSubmit($event)">
+      <form *ngIf="!checkCookie()" [formGroup]="submitForm" (ngSubmit)="onSubmit($event)">
         <div class="input-block">
           <input
             *ngIf="edited"
-            formControlName="email"
+            formControlName="user_email"
             placeholder="type your email here"
           />
           <span *ngIf="invalidEmail(submitForm)">
```

Comment:
negative names always confusing

rename function to isValidEmail()

and replace condition *ngIf="!isValidEmail(email)"